src/rabbit_vhost.erl

```diff
@@ -321,32 +339,38 @@ msg_store_dir_base() ->
 
 infos(Items, X) -> [{Item, i(Item, X)} || Item <- Items].
 
-i(name,    VHost) -> VHost;
-i(tracing, VHost) -> rabbit_trace:enabled(VHost);
-i(cluster_state, VHost) -> vhost_cluster_state(VHost);
-i(Item, _)        -> throw({bad_argument, Item}).
-
--spec info(rabbit_types:vhost()) -> rabbit_types:infos().
-
-info(VHost)        -> infos(?INFO_KEYS, VHost).
-
--spec info(rabbit_types:vhost(), rabbit_types:info_keys())
-                -> rabbit_types:infos().
+i(name,    VHost) -> ?vhost_field_name(VHost);
```

Comment:
This is confusing:
* The old `i(name, VHost)` returned its second argument, meaning it was the name of the vhost, not the record. This is confirmed by the `-spec()` of the old `infos()` function which took a `rabbit_types:vhost()`.
* The new `i()` takes a record now.